pvlib/clearsky.py

```diff
@@ -597,7 +599,258 @@ def _calc_d(aod700, p):
     return d
 
 
-def detect_clearsky(measured, clearsky, times, window_length,
+def bird(zenith, airmass_relative, aod380, aod500, precipitable_water,
```

Comment:
I'd prefer to see this function stay at the end of the module for this PR. The changes in this PR are not trivial so it helps to minimize the number of lines changed. I'm fine with a follow up PR to reorganize the module to your liking.